### ISSUE TITLE
Option to pretty print error log

### ISSUE THREAD

**Issue by BrunnerLivio** [nestjs/terminus#1925]
### Is there an existing issue that is already proposing this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe it

The JSON which is printed during an error message is not really readable.

### Describe the solution you'd like

Better readability of the current health check status. Example:

![terminus-log](https://user-images.githubusercontent.com/9899423/178226724-569242ba-2ed6-4942-aba6-9e0335842027.PNG)

This shall be enabled via:

```typescript
Terminus.forRoot({ errorLogStyle: 'pretty-emoji' | 'pretty' | 'json' })
```

This would allow one to switch between different log styles depending on the environment

```typescript
Terminus.forRoot({ errorLogStyle: process.env.NODE_ENV === 'production' ? 'json' : 'pretty-emoji' })
```
 

### Teachability, documentation, adoption, migration strategy

The `json`-option would remain the default in case someone uses a log parser tool (e.g. Splunk) to not break any changes.

The docs should recommend to use the code-snippet above. (`json` in prod but `pretty-emoji` locally)

### What is the motivation / use case for changing the behavior?

The JSON which is printed during an error message is not really readable.

**Comment by BrunnerLivio** [comment]
Note to myself:

The following code produces the result above (in `health-check.service.ts`)

```typescript
    printIndent (level) {
        if(level === 0) {
            return '';
        }
        return `${' '.repeat(level * 2)}- `;
    }
    
    printIndicatorSummary (result, level = 1) {
        let message = ''
        for(const [key, value] of Object.entries(result)) {
            if(typeof value === 'object' && value !== null) {
                message += `${this.printIndent(level)}${key}:\n${this.printIndicatorSummary(value, level + 1)}\n`;
            } else {
                message += `${this.printIndent(level)}${key}: ${value}\n`
            }
        }
        return message;
    }
    
    printSummary (result){
        let message = '';
    
        for(const [key, value] of Object.entries(result)) {
            if(value.status === "up") {
                message += `\x1b[0m\x1b[32m${key}: ✅\n${this.printIndicatorSummary(value)}\x1b[0m\n`
            }
            if(value.status === "down") {
                message += `\x1b[0m\x1b[31m${key}: ❌\n${this.printIndicatorSummary(value)}\x1b[0m\n`
            }
        }
        return message;
    }

    /**
     * Logs an error message of terminus
     * @param message The log message
     * @param error The error which was thrown
     */
    logError(message, causes) {
        message = `${message}\n\n${this.printSummary(causes)}`;
        this.logger.error(message);
    }
```

**Comment by BrunnerLivio** [comment]
Released with [9.1.0](https://github.com/nestjs/terminus/releases/tag/9.1.0) :tada:
